api/rpc/account/account.proto

```diff
@@ -246,3 +248,78 @@ message PermissionRequest {
   string level = 3;
   string resource_id = 4;
 }
+
+// Storage Types
+//This is the storage object
+
+message AccountEntry {
+  int64  id=1;
+  string name =2;
+  bool  is_user=3; // (true=user, false=org)
```

Comment:
The default value of false should not imply the account is an org. I prefer a protobuf enum for this:

enum type {
  USER = 0;
  ORGANIZATION = 1;
}